acceptance/auth/bundle_and_profile/test.toml

```diff
@@ -0,0 +1 @@
+Badness = "When -p flag is used inside the bundle folder for any CLI commands, CLI use bundle host anyway instead of profile one"
```

Comment:
How do I know from the output which host is used? The commands only output the user.

Reply:
Using correct host meaning current-user me returns a expected user. There are no CLI commands (API) which returns host being used, so we can't really use server stubs unless we deviate from API responses. 
We could use `describe auth describe` command to see the host but this would deviate a bit from idea of this test: "I want my autogenerated CLI commands to make calls against correct host depending on flags I pass"